src/app/contact/contact.component.html

```diff
@@ -14,7 +14,7 @@ <h5>Questions? Concerns? Press requests?</h5>
           Tweet us.</a>
       </div>
       <div class="col-sm-5">
-        <a href="http://www.kcwomenintech.org" target="_blank" class="kcwit-logo">
+        <a href="https://kcwomenintech.org/" target="_blank" class="kcwit-logo">
```

Comment:
Would it be possible to remove this? I think the problem is that you need to fetch latest from upstream, merge it into your origin, and rebase your branch off of it?

Reply:
I did it!! 😭